packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -429,7 +431,37 @@ void main() {
         await tester.pumpAndSettle();
 
         expect(controller.value.isPlaying, isFalse);
-        expect(controller.value.position, controller.value.duration);
+        expect(controller.value.position, nonzeroDuration);
```

Comment:
Previously, `duration` was zero so this assertion would pass even if `completed` didn't make any change to `position`.